### ISSUE TITLE
Implement objectively working Axe-core plugin

### ISSUE THREAD

**Issue by sergiobagd** [TourmalineCore/pelican-ui#126]
Я, как ***QA-инженер***
***Хочу*** внедрить надежный плагин axe-core, который будет использоваться в нашем проекте в автотестах
***Чтобы*** убедиться, что он работает корректно и объективно показывает имеющиеся несоответствия критериям доступности WCAG. 

## Описание
На данном этапе наш плагин axe-core игнорирует имеющиеся несоответствия критериям, которые откровенно не были покрыты - к примеру, создаем элемент и не присваиваем ему aria-label, а axe не ругается и ничего об этом не говорит

### Axe-core:
  - Видит любое расхождение
  
### II. Если система VIOLATIONS AXE-CORE (Имеет под капотом список критериев доступности, которые либо PASSED либо FAILED):

1) если >= 5 расхождений - TEST FAILED + REPORT AXE-CORE, 
2) если < 5 расхождений - TEST PASSED + REPORT AXE-CORE, 
3) если 0 расхождений -  TEST PASSED + "No violations found." MESSAGE

### III. Если система LIGHTHOUSE SCORE - 
1) если < 85 баллов - TEST FAILED + REPORT, 
2) если >= 85 баллов - TEST PASSED + отчет, 
3) если 100 баллов - TEST PASSED + "No violations found." MESSAGE

### Проблемы

1) Пайплайны 
- Блочить пайплайн, если A11Y != PERFECT - палки в колеса? (штук 6 violations, но по мелочи)

2) Критичность 
Критичность расхождений -> КОЛ-ВО МАЛО, НО КРИТИЧНОСТЬ ВАЖНАЯ??? 

Что делать тогда?
